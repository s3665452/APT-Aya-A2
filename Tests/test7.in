BUBLRBUYRYYRRRULYRRRYUURRRLRUYYBLUYRRRLRYYURRYYYULBYRYUYLLLYBUYYYUBBLBUUURULLLBBUBUBUBULLLBBBLBLBLBL
A1
B1
turn 1 B 2
turn 5 R 3
turn 0 Y 1
turn 3 Y 2
turn 0 R 4
turn 4 L 1
turn 2 U 3
turn 0 L 4
turn 0 U 3
turn 0 R 5
turn 0 Y 5
turn 0 B 6
turn 3 Y 5
turn 1 R 5
turn 5 R 3
turn 4 R 5
turn 0 Y 5
turn 0 L 4
turn 2 R 4
turn 0 B 1
turn 0 L 1
turn 0 U 3
turn 5 L 3
turn 3 L 4
turn 0 U 2
turn 2 Y 5
turn 1 U 2
turn 4 Y 5
turn 0 Y 4
turn 0 U 1
turn 0 R 5
turn 0 B 2

