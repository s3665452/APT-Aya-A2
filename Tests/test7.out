Factories:
0: F
1: B U Y Y
2: Y U B B
3: L B U U
4: U R U L
5: L L B B

Score for Player A1: 19
Mosaic for A1:
1:         . || . Y . . L
2:       . . || . B . . U
3:     . . . || U L . . R
4:   . . . . || R . . . Y
5: . . R R R || Y . . . .
broken:

Score for Player B1: 9
Mosaic for B1:
1:         . || B . . U L
2:       . B || . . Y . .
3:     . . . || U . . . R
4:   . . . . || . . L . .
5: . . . . . || Y R . . .
broken:

